Unrecognized class file version

Wanted to give wormhole a try but I bet the following error:
```
* What went wrong:
A problem occurred evaluating project ':app'.
> com/jakewharton/wormhole/gradle/GroovyVoodoo has been compiled by a more recent version of the Java Runtime (class file version 57.0), this version of the Java Runtime only recognizes class file versions up to 55.0
```
Would it be possible to lower the required Java version?

Thanks a lot for the lib!

After I upgraded to JDK13 I get the following error:
```
* What went wrong:
A problem occurred evaluating project ':app'.
> /usr/local/share/android-sdk/platforms/android-wormhole-0.2.0-android-29-r8-1.6.67
```
with the following stacktrace:
```
Caused by: java.nio.file.FileAlreadyExistsException: /usr/local/share/android-sdk/platforms/android-wormhole-0.2.0-android-29-r8-1.6.67
  at com.jakewharton.wormhole.gradle.AndroidWormhole$Companion.create(AndroidWormhole.kt:66)
  at com.jakewharton.wormhole.gradle.AndroidWormhole$Companion.create(AndroidWormhole.kt:42)
  at com.jakewharton.wormhole.gradle.AndroidWormhole$apply$1$voodoo$1.invoke(AndroidWormhole.kt:32)
  at com.jakewharton.wormhole.gradle.AndroidWormhole$apply$1$voodoo$1.invoke(AndroidWormhole.kt:24)
  at com.jakewharton.wormhole.gradle.AndroidWormholeKt$sam$java_util_function_Function$0.apply(AndroidWormhole.kt)
  at java_util_function_Function$apply.call(Unknown Source)
  at com.jakewharton.wormhole.gradle.GroovyVoodoo$_applyTo_closure1.doCall(GroovyVoodoo.groovy:23)
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  at org.gradle.internal.metaobject.BeanDynamicObject$MetaClassAdapter.invokeMethod(BeanDynamicObject.java:479)
  at org.gradle.internal.metaobject.BeanDynamicObject.tryInvokeMethod(BeanDynamicObject.java:191)
  at org.gradle.internal.metaobject.CompositeDynamicObject.tryInvokeMethod(CompositeDynamicObject.java:98)
  at org.gradle.internal.extensibility.MixInClosurePropertiesAsMethodsDynamicObject.tryInvokeMethod(MixInClosurePropertiesAsMethodsDynamicObject.java:3
  at org.gradle.internal.metaobject.ConfigureDelegate.invokeMethod(ConfigureDelegate.java:56)
  at build_4kp0ewgfoa9v8wpsvls2gjaly$_run_closure2.doCall(/Users/andre/projects/Internal-Cosmos/android-actyxos-app/app/build.gradle:22)
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  at org.gradle.util.ClosureBackedAction.execute(ClosureBackedAction.java:71)
  at org.gradle.util.ConfigureUtil.configureTarget(ConfigureUtil.java:154)
  at org.gradle.util.ConfigureUtil.configure(ConfigureUtil.java:105)
  at org.gradle.util.ConfigureUtil$WrappedConfigureAction.execute(ConfigureUtil.java:166)
  at org.gradle.internal.extensibility.ExtensionsStorage$ExtensionHolder.configure(ExtensionsStorage.java:173)
  at org.gradle.internal.extensibility.ExtensionsStorage.configureExtension(ExtensionsStorage.java:64)
  at org.gradle.internal.extensibility.DefaultConvention.configureExtension(DefaultConvention.java:372)
  at org.gradle.internal.extensibility.DefaultConvention.access$500(DefaultConvention.java:44)
  at org.gradle.internal.extensibility.DefaultConvention$ExtensionsDynamicObject.tryInvokeMethod(DefaultConvention.java:309)
  at org.gradle.internal.metaobject.CompositeDynamicObject.tryInvokeMethod(CompositeDynamicObject.java:98)
  at org.gradle.internal.extensibility.MixInClosurePropertiesAsMethodsDynamicObject.tryInvokeMethod(MixInClosurePropertiesAsMethodsDynamicObject.java:3
  at org.gradle.groovy.scripts.BasicScript$ScriptDynamicObject.tryInvokeMethod(BasicScript.java:134)
  at org.gradle.internal.metaobject.AbstractDynamicObject.invokeMethod(AbstractDynamicObject.java:162)
  at org.gradle.groovy.scripts.BasicScript.invokeMethod(BasicScript.java:83)
  at build_4kp0ewgfoa9v8wpsvls2gjaly.run(/Users/andre/projects/Internal-Cosmos/android-actyxos-app/app/build.gradle:21)
  at org.gradle.groovy.scripts.internal.DefaultScriptRunnerFactory$ScriptRunnerImpl.run(DefaultScriptRunnerFactory.java:91)
  ... 129 more
```

deleting `` gives

--- a/build.gradle
+++ b/build.gradle
@@ -31,6 +31,11 @@ subprojects {
     }
   }
 
+  plugins.withType(JavaBasePlugin) {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+  }
+
   repositories {
     mavenCentral()
     google()

